test/images/scripts/rhel-7.setup

```diff
@@ -84,6 +84,15 @@ sos \
 tuned \
 "
 
+# If the RHEL version is new enough, install setroubleshoot-server
```

Comment:
Wouldn't we just change this once our main image is new enough? It's not like we'll need to go back and create older images again.